libcontainer/factory_linux.go

```diff
@@ -256,11 +256,13 @@ func (l *LinuxFactory) StartInitialization() (err error) {
 		if _, ok := i.(*linuxStandardInit); ok {
```

Comment:
I'm actually not sure. If the `recover` code is doing what I think it's doing then we are effectively just `exit(1)`-ing (if you look at `main_unix.go` for runC).